### DISCUSSION TITLE
What's the difference between the batch table and the tags table?

### DISCUSSION THREAD

**Discussion by johnnyhchen** [gp-seattle/rms-app#158] [Backend Questions] [answered]
What's the difference between the batch table and the tags table? Just wondering what information is stored in each

**Comment by yauj** [comment, marked as answer]
Batch is groups of items (i.e. you can have a batch of items 123, 124 and 125)

Tags is more for searching purposes (i.e. items 4, 5 and 6 are associated to the 'Camera' tag)

**Reply by johnnyhchen** [reply]
Thank you!
Wanted to follow up and ask **what's the use cases of the Batch?**
Are batches pre-existing groups of items you can checkout at once? Or are they created upon checkout of a group of items then deleted? 
We wanted to know because we're trying to decide whether a "ReserveBatch()" function makes sense to create for the reservation system.

**Reply by yauj** [reply]
Example of a Batch would be `A2F Tech Equipment` with all the bins recorded under that.

**Reply by yauj** [reply]
Keep the reserve system simple for now though. Don't interact with borrow and return APIs.
